src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/client/UserProfileRequestHandlerTest.java

```diff
@@ -159,7 +156,7 @@ public <T> T sendDelete(MockMvc mockMvc,
                 .headers(getMultipleAuthHeaders())
                 .content(objectMapper.writeValueAsString(body))
                 .contentType(APPLICATION_JSON))
-                .andExpect(status().is(expectedHttpStatus.value())).andReturn();
```

Comment:
any reason for removing this? Can you point the code where you are really asserting the http status value?

Reply:
As discussed yesterday, We should not validate status in sendPost method. Send post method will return response and we should add assertion after getting response from it

Reply:
thanks @shivajihmcts . That's why I asked the second question Can you point the code where you are really asserting the http status value? . Have you changed all the code which calling this method and added assertion?Also, can you please remove the variable expectedHttpStatus as it is no longer used